crates/control/.cargo/config

```diff
@@ -0,0 +1,8 @@
+# Match build args from the workspace cargo config
+[build]
+rustflags = ["-C", "link-arg=-fuse-ld=lld"]
```

Comment:
Just making sure I understand why this is needed. Does this file completely override the top level .cargo/config?

Reply:
I added this when I thought it was the source of recompilation issues in CI. I *think* that Cargo looks for a config file in the hierarchy, but isn't necessarily merging them together. When I add/remove this line and run the build from inside `crates/control`, it rebuilds all the dependencies. :grimacing: To double check this, I did a `cargo clean` and then `cargo build` within `crates/control` with and then without these lines. Afterwards I end up with `target/debug` and `target/x86_64-unknown-linux-gnu/debug`. :sob: 

In summary, this is another sharp edge of workspaces. If you are working at the workspace root and also within a specific crate, you'll end up getting different compilation artifacts.